src/utils/envVars.ts

```diff
@@ -7,7 +7,10 @@ export const ENABLE_RPC_FEATURES        = Boolean(INFURA_PROJECT_ID && INFURA_PR
 export const INFURA_URL                 = `https://${IS_MAINNET ? "mainnet" : "goerli"}.infura.io/v3/${INFURA_PROJECT_ID}`;
 
 // public
-export const ETH2_NETWORK_NAME          = process.env.REACT_APP_ETH2_NETWORK_NAME   || 'Mainnet';
+export const TESTNET_LAUNCHPAD_NAME      = 'Pyrmont'
+export const TESTNET_LAUNCHPAD_URL      = `https://${TESTNET_LAUNCHPAD_NAME.toLowerCase()}.launchpad.ethereum.org/`
+export const MAINNET_LAUNCHPAD_URL      = 'https://launchpad.ethereum.org/'
+export const ETH2_NETWORK_NAME          = IS_MAINNET ? TESTNET_LAUNCHPAD_NAME : 'Mainnet';
```

Comment:
I think this is backwards, shouldn't it be:
```suggestion
export const ETH2_NETWORK_NAME          = IS_MAINNET ? 'Mainnet': TESTNET_LAUNCHPAD_NAME ;
```